### PR TITLE
Add az vaccine scraper

### PR DESCRIPTION
**Pull request by ghop02**
Adds in AZ vaccine scraper

### CODE REVIEW

**Comment by sglyon** [comment]
Hey @ghop02 I made some changes to the TableauMapClick class -- mostly cleaning up some of the surgical nested json queries using jmespath and renaming some variables.

I'm good with everything in here when you are, do you mind taking a look?

File: can_tools/scrapers/official/AZ/AZ_TableauPull.py [hunk 1]
```diff
@@ -157,3 +131,9 @@ def normalize(self, data):
             "value",
         ]
         return out.loc[:, cols_to_keep]
+
+
+if __name__ == "__main__":
+    d = ArizonaData()
+    data = d.fetch()
+    df = d.normalize(data)
```
Comment (ghop02) [inline can_tools/scrapers/official/AZ/AZ_TableauPull.py:139]: not sure if you want to make this a command line tool - i'm finding myself do things like:
```
python -m can_tools.scraper_maintenance latest-vaccine-data LAVaccineCounty  -d 1
```
all the time now :)

realize that this is a slight variation on that

File: can_tools/scrapers/official/base.py [hunk 2]
```diff
@@ -712,68 +718,42 @@ def extractTbluData(self, htmdump, area) -> pd.DataFrame:
...
-            "presModelHolder"
-        ]["genDataDictionaryPresModel"]["dataSegments"]["0"]["dataColumns"][0][
-            "dataValues"
-        ]
-        rlDat = htmdump["secondaryInfo"]["presModelMap"]["dataDictionary"][
-            "presModelHolder"
-        ]["genDataDictionaryPresModel"]["dataSegments"]["0"]["dataColumns"][1][
-            "dataValues"
-        ]
+        data_segment = jmespath.search(
```
Comment (ghop02) [inline can_tools/scrapers/official/base.py:723]: ooo that's cool

**Review by ghop02 — COMMENTED** [review]
This looks good to me! Can't approve my own PR, but I say ship it!

File: can_tools/scrapers/official/AZ/AZ_TableauPull.py (hunk 1 above)
Comment (ghop02) [reply can_tools/scrapers/official/AZ/AZ_TableauPull.py:139]: but I think whatever that looks like could/probably should be in another pr

Comment (sglyon) [reply can_tools/scrapers/official/AZ/AZ_TableauPull.py:139]: Ahh good catch, I forgot I didn't delete this one :laughing:

**Pull request merged**